book-a-reading-room-visit.web/Views/Home/Index.cshtml

```diff
@@ -15,7 +15,7 @@
             <div class="row">
                 <div class="col-md-12">
                     <p>
-                        Use this service to book a visit to view our documents. You may book a maximum of three visits in 4 weeks.<br>
+                        Use this service to book a visit to view our documents. You may book a maximum of four visits in 4 weeks.<br>
                         <a class="info-link" href="/book-a-reading-room-visit/about-the-book-a-reading-room-visit-service/">
```

Comment:
Any reason why we're using a mixture of words and numerals for numbers here? i.e. why not "three visits in four weeks" or "3 visits in 4 weeks"

Reply:
These are editorial change given by editors